doc_source/environment-variables.md

```diff
@@ -154,6 +154,9 @@ Accessing an environment secret during a build is similar to environment variabl
 
 You can use the following environment secrets that are accessible by default within the Amplify Console\.
 
+Example full SSM parameter name:
```

Comment:
The placement of this is confusing to me. Should it be under the table and say "Your full SSM parameter name should look something like this:"?